### PR TITLE
Added some new changes on Oh Ram Sam Sam song with instruction.

### PR DESCRIPTION
**Pull request by sylviawakoli**

### CODE REVIEW

**Comment by michaelclapham** [comment]
At 0:43 you have 
Gulli Gulli Gullli Ram Sam Sam
where it should be
Gulli Gulli Gulli Gulli Ram Sam Sam

**Comment by michaelclapham** [comment]
The instructions need to be marked as so, at the moment they're showing up as if a person is saying a lyric.
The first instruction is also wrong: 
Playing with the words and the sound.
should be
1. Play with the words and sounds.

**Comment by michaelclapham** [comment]
Sometimes when the same phrase is repeated e.g "Ooh Rafi Ooh Rafi Gulli Gulli Gulli Ram Sam Sam" at 2:23, instead of just showing the same text, it would be nice to add
Ooh Rafi Ooh Rafi Gulli Gulli Gulli Ram Sam Sam
(again)
to indicate that the same thing is being sung a second time.

**Comment by michaelclapham** [comment]
Also please include the name of the song in the title of the pull request in future! 😹

**Comment by estherwafula** [comment]
@sylviawakoli  You can change your user stories and song title by clicking on the edit button on the pull request

**Comment by maxwellfundi** [comment]
@sylviawakoli a quick one here- there some suggestions by @michaelclapham that you have not done

1. The instructions need to be marked with the column- isInstruction:TRUE; so that they are displayed properly
2.  at 0.9 you have got Ram- Should be Oh Ram...
3. The Guli Guli dont seem quite right at  0.13- One Gulli is missing  and also at 0.21 you have an extra Gulli- at 1.17  you also have a Gulli missing for the second person and and 1.30 you have a gulli missing for both persons. The gullis are mostly in fours so good to check and be keen on that. 
4.  From 1.56 they both sing together so the lyrics should have the same timings for both people

File: src/app/lyrics/songs/oh-ram-sam-sam-with-instructions.ts [hunk 1]
```diff
@@ -237,71 +237,71 @@ export const Oh_Ram_Sam_Sam_Instructions: Song = {
...
+            "lyric": "Ooh Rafi                          Ooh Rafi                       Gulli Gulli Gulli          Ram Sam Sam",
             "person": "1",
             "startTimeMinute": "1",
             "startTimeSecond": "29",
             "endTimeMinute": "1",
             "endTimeSecond": "34"
         },
         {
-            "lyric": "Add Fun Actions",
+            "lyric": "Three Add Fun Actions",
```
Comment (estherwafula) [inline src/app/lyrics/songs/oh-ram-sam-sam-with-instructions.ts:304]: Add the word Instruction:before the following words,Three add fun actions

**Comment by maxwellfundi** [comment]
@michaelclapham I think this should be looking alot better now.

**Comment by michaelclapham** [comment]
Chiara will have a look at this one.

**Comment by fagiothree** [comment]
[oh-ram-sam-sam-with-instructions.txt](https://github.com/michaelclapham/mhac-app-frontend/files/4621171/oh-ram-sam-sam-with-instructions.txt

**Comment by sylviawakoli** [comment]
I made all the necessary changes needed

**Comment by maxwellfundi** [comment]
Alright i am merging this now :-)

**Pull request merged**